### PR TITLE
docs(cli): better describe actions

### PR DESCRIPTION
**Pull request by uguy**
Explain how actions are applied when configuring several for a key.
Make action title upper case and remove description for title.

Closes https://bonitasoft.atlassian.net/browse/BON-337

### CODE REVIEW

File: modules/cli/pages/actions-anonymization.adoc [hunk 1]
```diff
@@ -1,34 +1,59 @@
...
+[source,yaml]
+----
+  arch_flownode_instance:
+      displayname:
+        actions:
+          - action: REPLACE_WITH_OTHER
+            value: description
+          - action: REGEX_REPLACE
+            pattern: The task Allocate repair agent on car (\S+) (is now assigned to .*)
+            value: The task Allocate repair agent on car *** $2
```
Comment (tbouffard) [inline modules/cli/pages/actions-anonymization.adoc]: **thought:** I'm not happy with the example I've chosen.
It's not functionally meaningful. We should find a simple regexp that could be used here for the case of a description.

File: modules/cli/pages/actions-anonymization.adoc [hunk 2]
```diff
@@ -1,34 +1,59 @@
...
+If no actions are explicitly specified in the configuration for a particular value, it will default to a predefined action as described in the page
+xref:configuration-for-anonymization.adoc[Configure the anonymization].
 
-Additionally, you can define a fallback mechanism in case the primary action does not work or does not find a match (for example, using the `regex_replace` action).
+Additionally, you can define a fallback mechanism in case the primary action does not work or does not find a match (for example, using the `REGEX_REPLACE` action).
 
-== Hash - Hash a value
+=== Order of execution of the actions
+
+The actions are executed in the order they are defined in the configuration file. The order of the actions is important because the output of one action is the input of the next action.
```
Comment (tbouffard) [inline modules/cli/pages/actions-anonymization.adoc:15]: **thought:** we should also explain how the fallback interact with the list of actions

File: modules/cli/pages/actions-anonymization.adoc [hunk 3]
```diff
@@ -1,34 +1,59 @@
...
-Action types are mechanisms used to anonymize specific values in the exported data. A value can have one or multiple actions associated with it. If no actions are explicitly specified in the configuration for a particular value, it will default to a predefined action as described in the page xref:configuration-for-anonymization.adoc[Configure the anonymization].
+Action types are mechanisms used to anonymize specific values in the exported data. A value can have one or multiple actions associated with it.
+If no actions are explicitly specified in the configuration for a particular value, it will default to a predefined action as described in the page
+xref:configuration-for-anonymization.adoc[Configure the anonymization].
 
-Additionally, you can define a fallback mechanism in case the primary action does not work or does not find a match (for example, using the `regex_replace` action).
+Additionally, you can define a fallback mechanism in case the primary action does not work or does not find a match (for example, using the `REGEX_REPLACE` action).
 
-== Hash - Hash a value
+=== Order of execution of the actions
```
Comment (tbouffard) [inline modules/cli/pages/actions-anonymization.adoc:13]: **thought:** we should also mention the special case that occurs with `REPLACE_WITH_OTHER`

Comment (passga) [reply modules/cli/pages/actions-anonymization.adoc:13]: Not finally there is not particular things for replace_with_other, the order wil be same as defined in configuration file.

Comment (uguy) [reply modules/cli/pages/actions-anonymization.adoc:13]: replace_with_other are applied in the end. Documented now.

**Review by tbouffard — APPROVED** [review]
LGTM

**Pull request merged**